1.5/examples/index.md

```diff
@@ -78,4 +78,4 @@ features of Apache Accumulo.
                        Also shows user permissions.
 
 
-[1]: {{ site.baseurl }}/1.5/accumulo_user_manual#_user_administration
+[1]: {{ site.baseurl }}/1.5/user-manual#_user_administration
```

Comment:
`/1.5/accumulo_user_manual` still goes to the stand-alone manual so the links won't be broken.  However, I could move rename the stand-alone manual and make `/1.5/accumulo_user_manual` the embedded manual (which is now `1.5/user-manual`).  This makes sense to me now that I the '#section` links to work in the embedded manual due to 681f1ac9421ccd30dedd5db550d67e9842cc0a52